### DISCUSSION TITLE
How to remove toolbar actions internal html tags

### DISCUSSION THREAD

**Discussion by Satsh24** [GrapesJS/grapesjs#5230] [General]
Hi team, with version(0.19.5) we have noticed that the internal HTML tags inside the main (mj-text) are now selectable/deletable and etc. This feature is actually breaking some of our other logic. Is there an easy way to disable these toolbars and highlighting of the internal HTML tags other mj-*?

can you please take a look @artf 

<img width="395" alt="image" src="https://github.com/GrapesJS/grapesjs/assets/40301607/59ca99d6-0428-4b18-a3f6-056a9894ecdb">

**Comment by artf** [comment]
I guess you need `disableTextInnerChilds`, added from this version https://github.com/GrapesJS/grapesjs/releases/tag/v0.21.2
